### ISSUE TITLE
Panel improvements

### ISSUE THREAD

**Issue by simurai** [joefitzgerald/go-plus#477]
> @joefitzgerald 

```
1) cmd-a in a tab with content then cmd-c = copies tab titles also
2) weird styling if using the material theme
3) And then a request to allow docking right. Not sure how I would display tabs in that model
4) Oh and one other oddity: for the panel that I have for test output (which sets innerhtml) the scroll bar occurs at the right boundary of the text (i.e. It seems the container width is not 100%)
```

**Comment by simurai** [comment]
> 1) cmd-a in a tab with content then cmd-c = copies tab titles also

If you add the `readonly` attribute to `panel-body`, then clicking text inside selects it and you only copy that text.

![copy](https://cloud.githubusercontent.com/assets/378023/19145565/7824e42a-8bea-11e6-94c6-67ab5d7b2a7d.gif)

But you can't select part of the text, so not sure that's a good idea. Here something about [createTextRange()](http://stackoverflow.com/questions/1173194/select-all-div-text-with-single-mouse-click) that might can be used instead.

**Comment by simurai** [comment]
2) Screenshot? I can see the text in the tabs being a bit big.

**Comment by simurai** [comment]
3) Hmm.. yeah. Just 2-3 tabs in a panel on the right should be fine. But if there are a lot, most will be hidden. DevTools adds a >> button, that when you click it, there is a dropdown to pick. But a lot of extra work.

**Comment by sadovnychyi** [comment]
![image](https://cloud.githubusercontent.com/assets/193864/19161079/f80a0fd2-8c24-11e6-9ef7-0eef33a4aa06.png) – is it possible to hide this button for non-Go grammars?

**Comment by joefitzgerald** [comment]
Yeah I think we could do that @sadovnychyi

**Comment by Ehekatl** [comment]
just get this panel after update, how could I disable it ? already get many panel that took a lot work space

**Comment by lloiser** [comment]
You cannot disable it, just hide it using the `X` in the top right of the panel. Also make sure to change the go-plus setting `Panel Display Mode` to `closed` so it won't open when starting atom.

**Comment by BafFlynN** [comment]
Is it possible to make the panel be available on the right side of the screen?

**Comment by joefitzgerald** [comment]
This is partially fixed in `go-plus` `v5.0.0`: the panel can now be vertical.

**Comment by joefitzgerald** [comment]
Closing this in deference to #518 and #519. The remaining portions of this issue are fixed.
